### DISCUSSION TITLE
Channels stuck closing after recovery.

### DISCUSSION THREAD

**Discussion by silverzimmermann** [lightningnetwork/lnd#8223] [Troubleshooting]
Hi,

I'm using Umbrel node and had to do clean install. After recovering my LND from seed and uploading backup, it initiated channels closure but only one channel closed successfully and other are stuck and show 0 local and remote balance.

This is the output of pendingchannels:

{
    "total_limbo_balance":  "0",
    "pending_open_channels":  [],
    "pending_closing_channels":  [],
    "pending_force_closing_channels":  [],
    "waiting_close_channels":  [
        {
            "channel":  {
                "remote_node_pub":  "02f1a8c87607f415c8f22c00593002775941dea48869ce23096af27b0cfdcc0b69",
                "channel_point":  "b677c55ba0de50f38c261609b8ea0acad36069db131f91c734bd3083771ed098:1",
                "capacity":  "1000000",
                "local_balance":  "0",
                "remote_balance":  "0",
                "local_chan_reserve_sat":  "0",
                "remote_chan_reserve_sat":  "0",
                "initiator":  "INITIATOR_LOCAL",
                "commitment_type":  "ANCHORS",
                "num_forwarding_packages":  "0",
                "chan_status_flags":  "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private":  true,
                "memo":  ""
            },
            "limbo_balance":  "0",
            "commitments":  {
                "local_txid":  "",
                "remote_txid":  "",
                "remote_pending_txid":  "",
                "local_commit_fee_sat":  "0",
                "remote_commit_fee_sat":  "0",
                "remote_pending_commit_fee_sat":  "0"
            },
            "closing_txid":  ""
        },
        {
            "channel":  {
                "remote_node_pub":  "035e4ff418fc8b5554c5d9eea66396c227bd429a3251c8cbc711002ba215bfc226",
                "channel_point":  "30bb0530df7db8b042f19bb617c3ea10742c76663b082e6cc4b60e2be60108e8:1",
                "capacity":  "5000000",
                "local_balance":  "0",
                "remote_balance":  "0",
                "local_chan_reserve_sat":  "0",
                "remote_chan_reserve_sat":  "0",
                "initiator":  "INITIATOR_LOCAL",
                "commitment_type":  "ANCHORS",
                "num_forwarding_packages":  "0",
                "chan_status_flags":  "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private":  true,
                "memo":  ""
            },
            "limbo_balance":  "0",
            "commitments":  {
                "local_txid":  "",
                "remote_txid":  "",
                "remote_pending_txid":  "",
                "local_commit_fee_sat":  "0",
                "remote_commit_fee_sat":  "0",
                "remote_pending_commit_fee_sat":  "0"
            },
            "closing_txid":  ""
        },
        {
            "channel":  {
                "remote_node_pub":  "0364913d18a19c671bb36dd04d6ad5be0fe8f2894314c36a9db3f03c2d414907e1",
                "channel_point":  "658b18668aac5e3d3ca00c15ac7495f2cfee4f16700846c33243ab1966fb9c2d:8",
                "capacity":  "2000000",
                "local_balance":  "0",
                "remote_balance":  "0",
                "local_chan_reserve_sat":  "0",
                "remote_chan_reserve_sat":  "0",
                "initiator":  "INITIATOR_REMOTE",
                "commitment_type":  "ANCHORS",
                "num_forwarding_packages":  "0",
                "chan_status_flags":  "ChanStatusRestored",
                "private":  true,
                "memo":  ""
            },
            "limbo_balance":  "0",
            "commitments":  {
                "local_txid":  "",
                "remote_txid":  "",
                "remote_pending_txid":  "",
                "local_commit_fee_sat":  "0",
                "remote_commit_fee_sat":  "0",
                "remote_pending_commit_fee_sat":  "0"
            },
            "closing_txid":  ""
        },
        {
            "channel":  {
                "remote_node_pub":  "03e691f81f08c56fa876cc4ef5c9e8b727bd682cf35605be25d48607a802526053",
                "channel_point":  "60314a9b5a0863d9a8b581cf00ad07e560541681527db2f5b8426b865e5820f2:0",
                "capacity":  "20000",
                "local_balance":  "0",
                "remote_balance":  "0",
                "local_chan_reserve_sat":  "0",
                "remote_chan_reserve_sat":  "0",
                "initiator":  "INITIATOR_LOCAL",
                "commitment_type":  "ANCHORS",
                "num_forwarding_packages":  "0",
                "chan_status_flags":  "ChanStatusLocalDataLoss|ChanStatusRestored",
                "private":  true,
                "memo":  ""
            },
            "limbo_balance":  "0",
            "commitments":  {
                "local_txid":  "",
                "remote_txid":  "",
                "remote_pending_txid":  "",
                "local_commit_fee_sat":  "0",
                "remote_commit_fee_sat":  "0",
                "remote_pending_commit_fee_sat":  "0"
            },
            "closing_txid":  ""
        }
    ]
}

**Comment by guggero** [comment]
I did a quick check and it looks like the nodes are all still around. So they should be closing your channels. But with the crazy mempool fees it might actually take a while for them to close. So you just need to be patient and hope for the mempool to get a bit less crazy soon.
